docs/user-guide/admin-cli.md

```diff
@@ -16,389 +16,447 @@ The admin cli also enables ESX admins to implement tenancy.
-Uuid                                  Name      Description               Default_datastore  VM_list
-------------------------------------  --------  ------------------------  -----------------  -------
-527e94ec-43e9-4d78-81fe-d99ab06a54b3  _DEFAULT  This is a default tenant
+vm-group rm succeeded
+
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vm-group ls
+Uuid                                  Name      Description                 Default_datastore  VM_list
+------------------------------------  --------  --------------------------  -----------------  -------
+11111111-1111-1111-1111-111111111111  _DEFAULT  This is a default vm-group
 ```
 
 #### Help
 ```
-[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant rm -h
-usage: vmdkops_admin.py tenant rm [-h] --name NAME [--remove-volumes]
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vm-group rm -h
+usage: vmdkops_admin.py vm-group rm [-h] --name NAME [--remove-volumes]
 
 optional arguments:
   -h, --help        show this help message and exit
-  --name NAME       The name of the tenant
-  --remove-volumes  BE CAREFUL: Removes this tenant volumes when removing a
-                    tenant
+  --name NAME       The name of the vm-group
+  --remove-volumes  BE CAREFUL: Removes this vm-group volumes when removing a
+                    vm-group
 
 ```
 
 ### Virtual Machine
 
 #### Add
-Add a VM to a tenant. A VM can only access the datastores for the tenant it is assigned to.
-VMs can be assigned to only one tenant at a time.
+Add a VM to a vm-group. A VM can only access the datastores for the vm-group it is assigned to.
+VMs can be assigned to only one vm-group at a time.
 ```
-[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant vm add --name=tenant1 --vm-list=photon5
-tenant vm add succeeded
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vm-group ls
+Uuid                                  Name       Description                 Default_datastore  VM_list
+------------------------------------  ---------  --------------------------  -----------------  --------
+11111111-1111-1111-1111-111111111111  _DEFAULT   This is a default vm-group
+6d810c66-ffc7-47c8-8870-72114f86c2cf  vm-group1                                                 photon-6
+
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vm-group vm add --name=vm-group1 --vm-list=photon7
+vm-group vm add succeeded
+
 ```
 
 #### List
 ```
-[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant vm ls --name=tenant1
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vm-group vm ls --name=vm-group1
 Uuid                                  Name
-------------------------------------  -------
-564df562-3d58-c99a-e76e-e8792b77ca2d  photon4
-564d4728-f1c7-2029-d01e-51f5e6536cd9  photon5
+------------------------------------  --------
+564d5849-b135-1259-cc73-d2d3aa1d9b8c  photon-6
```

Comment:
Can we keep the name consistent? Either photon-6 and photon-7, or photon6 and photon7. Same for other occurrences.